### PR TITLE
add set/get tests for remaining UdpSocket setters

### PR DESCRIPTION
**Pull request by dtacalau**
Signed-off-by: Daniel Tacalau <dst4096@gmail.com>

### CODE REVIEW

File: tests/udp_socket.rs [hunk 1]
```diff
@@ -140,6 +140,92 @@ fn get_ttl_without_previous_set() {
...
+    assert_eq!(socket1.broadcast().unwrap(), false);
+
+    assert!(socket1.take_error().unwrap().is_none());
+}
+
+#[test]
+fn get_broadcast_without_previous_set() {
+    let socket1 = UdpSocket::bind(any_local_address()).unwrap();
+
+    socket1
```
Comment (Thomasdezeeuw) [inline tests/udp_socket.rs]: Want assert its false?

File: tests/udp_socket.rs [hunk 2]
```diff
@@ -140,6 +140,92 @@ fn get_ttl_without_previous_set() {
...
+    assert_eq!(socket1.multicast_loop_v4().unwrap(), false);
+
+    assert!(socket1.take_error().unwrap().is_none());
+}
+
+#[test]
+fn get_multicast_loop_v4_without_previous_set() {
+    let socket1 = UdpSocket::bind(any_local_address()).unwrap();
+
+    socket1
```
Comment (Thomasdezeeuw) [inline tests/udp_socket.rs]: Again want to assert it false?

File: tests/udp_socket.rs [hunk 3]
```diff
@@ -140,6 +140,92 @@ fn get_ttl_without_previous_set() {
...
+    assert_eq!(socket1.multicast_loop_v6().unwrap(), false);
+
+    assert!(socket1.take_error().unwrap().is_none());
+}
+
+#[test]
+fn get_multicast_loop_v6_without_previous_set() {
+    let socket1 = UdpSocket::bind(any_local_ipv6_address()).unwrap();
+
+    socket1
```
Comment (Thomasdezeeuw) [inline tests/udp_socket.rs]: Again; assert false?

File: tests/udp_socket.rs [hunk 4]
```diff
@@ -140,6 +140,92 @@ fn get_ttl_without_previous_set() {
...
+    assert_eq!(socket1.multicast_ttl_v4().unwrap(), TTL);
+
+    assert!(socket1.take_error().unwrap().is_none());
+}
+
+#[test]
+fn get_multicast_ttl_v4_without_previous_set() {
+    let socket1 = UdpSocket::bind(any_local_address()).unwrap();
+
+    socket1
```
Comment (Thomasdezeeuw) [inline tests/udp_socket.rs]: Again; assert false?

File: tests/udp_socket.rs (hunk 1 above)
Comment (dtacalau) [reply tests/udp_socket.rs]: No, just want to call the get w/o previous set. Plus, it's value might be platform dependent, and we'd have to check value based on platform.

File: tests/udp_socket.rs (hunk 2 above)
Comment (dtacalau) [reply tests/udp_socket.rs]: No, just call the get.

File: tests/udp_socket.rs (hunk 4 above)
Comment (dtacalau) [reply tests/udp_socket.rs]: No, just call the get.

File: tests/udp_socket.rs (hunk 1 above)
Comment (Thomasdezeeuw) [reply tests/udp_socket.rs]: Fair enough.

**Comment by dtacalau** [comment]
> LGTM, blocked on #1192.

Rebased on master after #1192 merge.

**Pull request merged**